stock_picking_transfer_lot_autoassign/models/stock_picking.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
```

Comment:
@rgarnau Sorry, missed this : remove api

Reply:
done :+1: